spec/features/jobs/user_edits_a_job_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+describe "user edits an existing job" do
+  scenario "a user can edit a job" do
+    company = Company.create!(name: "StackOverflow")
+    job = company.jobs.create!(title: "Rails Developer",
+      description: "do rails things", level_of_interest: 3, city: "Denver")
+
+    visit edit_company_job_path({id: job.id, company_id: company.id})
+    fill_in("job[title]", with: "Backend Engineer")
+    fill_in("job[description]", with: "do database things")
+    click_on("Update Job")
+
+    expect(current_path).to eq(company_job_path(company, job))
+
+    within(".title") do
+      expect(page).to have_content("Backend Engineer")
+    end
+
+    within(".description") do
+      expect(page).to have_content("do database things")
+    end
```

Comment:
*Might* add an expectation that nothing changed that you didn't expect to change (e.g. the interest level is still 3, and the city is still Denver).